shell/browser/ui/inspectable_web_contents.cc

```diff
@@ -585,6 +587,27 @@ void InspectableWebContents::LoadCompleted() {
           prefs.FindString("currentDockState");
       base::RemoveChars(*current_dock_state, "\"", &dock_state_);
     }
+#if BUILDFLAG(IS_WIN)
+    auto* api_web_contents = api::WebContents::From(GetWebContents());
+    if (api_web_contents) {
+      auto* win =
+          static_cast<NativeWindowViews*>(api_web_contents->owner_window());
+      // When WCO is enabled, undock the devtools if the current dock
+      // position overlaps with the position of window controls to avoid
+      // broken layout.
+      if (win && win->IsWindowControlsOverlayEnabled()) {
+        if (IsAppRTL()) {
+          if (dock_state_ == "left") {
```

Comment:
nit: unless you think it's clearer to nest these ifs, I think it's a little simpler just to say:

```cpp
 if (IsAppRTL() && dock_state_ == "left") {
   dock_state_ = "undocked";
 } else if (dock_state_ == "right") {
   dock_state_ = "undocked";
 }
 ```